06.bowling_object/game.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require_relative 'frame'
+
+class Game
+  def initialize(marks)
+    @frames = []
+    mark_idx = 0
+    0.upto(Frame::LAST_FRAME) do |frame_idx|
+      frame = Frame.new(frame_idx)
+      mark_idx += frame.add_shot(marks[mark_idx..])
+      @frames << frame
```

Comment:
```suggestion
    @frames = 0.upto(Frame::LAST_FRAME).map do |frame_idx|
      frame = Frame.new(frame_idx)
      mark_idx += frame.add_shot(marks[mark_idx..])
      frame
```

mapの方がちょっとシンプルになりそうです。